### PR TITLE
Turn HTTPS back on for ocean staging

### PR DESCRIPTION
**Pull request by yuvipanda**
DNS has been set.

### CODE REVIEW

File: deployments/ocean/config/staging.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 jupyterhub:
   proxy:
     https:
-      enabled: false
       hosts:
       - ocean.pangeo.io
```
Comment (rabernat) [inline deployments/ocean/config/staging.yaml]: should this be `staging.ocean.pangeo.io`?

Comment (yuvipanda) [reply deployments/ocean/config/staging.yaml]: @rabernat aha, yes it should be!

**Comment by rabernat** [comment]
How do we ensure that the cluster stays at the same IP address? Does that need to go into the config?

**Comment by yuvipanda** [comment]
@rabernat yes! You need to do two things:

1. Set `jupyterhub.proxy.service.loadBalancerIP` in staging to current IP
2. Go to https://console.cloud.google.com/networking/addresses/list?folder=&organizationId=&project=pangeo-181919 and mark that IP as 'static' - so we don't lose it without explicitly deleting it.

This would make sure that we have access to this IP forever, and lets us switch hubs between clusters if we need to.

**Pull request merged**